ui/cryptomaterial/segmented_control.go

```diff
@@ -57,14 +59,20 @@ func (sc *SegmentedControl) Layout(gtx C) D {
 						txt.Color = sc.theme.Color.Text
 						border = Border{Radius: Radius(8)}
 					}
-					return LinearLayout{
+
+					ll := LinearLayout{
 						Width:      WrapContent,
 						Height:     WrapContent,
-						Padding:    layout.UniformInset(values.MarginPadding8),
 						Background: bg,
 						Margin:     layout.UniformInset(values.MarginPadding5),
 						Border:     border,
-					}.Layout2(gtx, txt.Layout)
+					}
+					if sc.Padding != nil {
+						ll.Padding = *sc.Padding
+					} else {
+						ll.Padding = layout.UniformInset(values.MarginPadding8)
+					}
```

Comment:
Use a default value. Eliminate pointer.

Reply:
The use of a pointer value in this method is to allow callers to ignore some config values.  ll.Padding is not exported so callers can't modify that directly.

Will take another look though.